### ISSUE TITLE
Unit Testing

### ISSUE THREAD

**Issue by ericraio** [react-native-simple-router-community/react-native-simple-router#58]
How should we handling unit testing this project? Should we include this as apart of the definition of done?

**Comment by davidLeonardi** [comment]
Yes please. Does anyone of you have experience with unittesting rn stuff?

**Comment by charpeni** [comment]
Totally !

> Does anyone of you have experience with unittesting rn stuff?

Not me :(

**Comment by ericraio** [comment]
I have a lot of experience testing JavaScript but not RN explicitly, I am sure it's not too hard. I should have some time tomorrow to take a look into this.

**Comment by davidLeonardi** [comment]
Same here, but I'd appreciate some input with someone who has some experience with RN specific testing.
Anybody of our users has some tried-and-tested setup they use professionally?

**Comment by ericraio** [comment]
@sethx Yeah I agree. I did find this however. https://gist.github.com/jmreidy/4145809229195441d4d4

**Comment by charpeni** [comment]
http://reactkungfu.com/2015/07/approaches-to-testing-react-components-an-overview/

**Comment by davidLeonardi** [comment]
/me thinks that react testing is different from react native testing.

I'd like to wait for some additional input from the community before making a decision.

**Comment by charpeni** [comment]
https://github.com/airbnb/enzyme/blob/master/docs/guides/react-native.md

**Comment by davidLeonardi** [comment]
That's interesting... will look at that over the weekend.
nice find!
